Source/YoutubeConfig.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  YoutubeConfig.swift
+//  edX
+//
+//  Created by Andrey Cañon on 14/9/168.
+//  Copyright © 2018 edX. All rights reserved.
+//
+
+import Foundation
+
+fileprivate enum YoutubeKeys: String, RawStringExtractable {
+    case Enabled = "ENABLED"
+    case YoutubeApiKey = "YOUTUBE_API_KEY"
+}
+
+class YoutubeConfig: NSObject {
```

Comment:
Should this class be called EmbeddedYoutubeConfig?

Reply:
based on another config files this should call YoutubeVideoConfig, because the flag is  
```
YOUTUBE_VIDEO:
    ENABLED: True
    YOUTUBE_API_KEY: 'YOUR_API_KEY'  
```